packages/components/organisms/f-checkout/src/components/ErrorDialog.vue

```diff
@@ -61,25 +76,41 @@ export default {
 
         serviceTypeText () {
             return this.$t(`serviceTypes.${this.serviceType}`);
+        },
+
+        isDuplicateOrderError () {
+            return this.errorCode === 'DuplicateOrder';
         }
     },
 
     mounted () {
         const modalContext = this.getModalContext();
 
-        if (modalContext) modalContext.open();
+        if (modalContext) {
+            modalContext.open();
+        }
+
+        if (this.isDuplicateOrderError) {
+            this.trackDuplicateOrderWarnDialog();
+        }
     },
 
     methods: {
         ...mapActions(VUEX_CHECKOUT_MODULE, [
             'updateMessage'
         ]),
 
+        ...mapActions(VUEX_CHECKOUT_ANALYTICS_MODULE, [
+            'trackDuplicateOrderWarnDialog'
+        ]),
+
         getModalContext () {
             const { errorModal } = this.$refs;
             const { megaModal } = errorModal.$refs;
 
-            if (megaModal) return errorModal;
+            if (megaModal) {
```

Comment:
I think this can stay as it was? `if (megaModal) return errorModal;`

Reply:
or

```suggestion
            return megaModal ? errorModal : null
```

🙂

Reply:
That looks cooler than mine 😂